emscripten.py

```diff
@@ -312,19 +310,26 @@ def update_settings_glue(settings, metadata):
   # Integrate info from backend
   if settings['SIDE_MODULE']:
```

Comment:
Forgot that it wasn't implemented, for one.

Mostly just, should we consolidate the setting of these settings. I see our options as:
1. leave this split into two functions, one of which supersets the other
2. combine these, but put the non-wasm-backend portions behind `if not is_wasm_backend():`, adding complexity to how the function executes
3. combine these, do possibly-irrelevant flag setting in the wasm-backend case, hope that the error is caught elsewhere

I guess what I'm saying is, 3 sounds least-bad to me, but it's a fairly weak preference, so I'm curious if anyone else has better ideas.